README.md

```diff
@@ -30,4 +25,17 @@ Example Assembly: https://a360.co/3RiKY86
 For the advanced division, both the vertical and radial position of a given leaf or flower can vary.
 Stems can be short, medium, or long, and can be positioned at one of five vertical positions.
 
+# Painting
+All surfaces are spray painted with colors as follows:
+
+- Stems - [Satin Moss Green](https://www.homedepot.com/p/Rust-Oleum-Painter-s-Touch-2X-12-oz-Satin-Moss-Green-General-Purpose-Spray-Paint-334075/307244920)
+- Healthy leaves - [Gloss Meadow Green](https://www.homedepot.com/p/Rust-Oleum-Painter-s-Touch-2X-12-oz-Gloss-Green-General-Purpose-Spray-Paint-334039/307244868)
+- Unhealthy leaves - [Satin Green Apple](https://www.homedepot.com/p/Rust-Oleum-Painter-s-Touch-2X-12-oz-Satin-Green-Apple-General-Purpose-Spray-Paint-334070/307244897)
+- Flowers - [Flat White](https://www.homedepot.com/p/Rust-Oleum-Painter-s-Touch-2X-12-oz-Flat-White-General-Purpose-Spray-Paint-334021/307244842)
 
+# Assembly Instructions
+1. Print an appropriate number of stalks, leaves, flowers, and stems (of various lengths) per the rules. If desired, CAD files may be modified as needed to achieve reliable press fits in the steps below (given your printer / material).

```

Comment:
Could you read the rules and suggest an appropriate number of each based on your understanding? (I'll check you.)